scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
+    uri = URL(DOMAIN+RANDOM)
```

Comment:
We might be able to do `URL.new(DOMAIN).join(RANDOM)` here, or something similar. It'd mean we join using the proper method …and this helps with the `/` issue you mentioned.